### DISCUSSION TITLE
Use sdk, how could i control timeout for many group scan?

### DISCUSSION THREAD

**Discussion by Jarnpher553** [projectdiscovery/nuclei#5314] [Q&A]
In my business, i hava many group scanning. When i use the sdk, i found the http client pool may be init once in program environment. So, does sdk has some way to control the per timeout for per group scanning in one program?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
